.github/actions/determine-binary-name/action.yml

```diff
@@ -0,0 +1,56 @@
+name: Determine the name of a binary
```

Comment:
If I understand we've pulled out some of the steps from `workflow-build` to be their own actions?

Reply:
Correct as these steps are used in multiple places.